app/assets/themes/fearlesscities/_footer.scss

```diff
@@ -0,0 +1,6 @@
+@import "variables";
+
+.mini-footer, .main-footer {
+  //background-image: image-url("fearlesscities/texture-blue.jpg");
```

Comment:
It would be cool to make sure that the same font color is used for every part of the footer (I think the black is poorly readable on top of the footer background, which is something that hasn't been considered in the [main page](https://fearlesscities.com/))